portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase

```diff
@@ -1372,165 +1520,42 @@ definition {
-			OAuth2.revokeGrantedAuthViaUser(applicationName = "OAuth Application");
-		}
-
-		task ("View granted auth via user has no authorizations") {
-			OAuth2.viewGrantedAuthViaUser(noAuthorizations = "true");
-		}
-	}
-
-	@description = "This is a use case for OAUTH2-216. View auth prompt requested scopes."
-	test ViewAuthPromptRequestedScopes {
-		property custom.properties = "virtual.hosts.default.site.name=Guest";
-		property osgi.module.configuration.file.names = "com.liferay.oauth2.provider.scope.internal.configuration.BundlePrefixHandlerFactoryConfiguration~default.config";
-		property osgi.module.configurations = "include.bundle.symbolic.name=&quot;false&quot;${line.separator}osgi.jaxrs.name=&quot;Default&quot;${line.separator}separator=&quot;&quot;${line.separator}service.properties=&quot;&quot;";
-		property portal.upstream = "quarantine";
-		property test.name.skip.portal.instance = "OAuth2#ViewAuthPromptRequestedScopes";
+	@description = "This is a use case for LPS-105158. Scopes display as list on instance level."
+	@priority = "4"
+	test ScopesShouldBeDisplayedAsListOnInstanceLevel {
+		property test.name.skip.portal.instance = "OAuth2#ScopesShouldBeDisplayedAsListOnInstanceLevel";
 
-		task ("Go to OAuth 2 administration") {
-			ApplicationsMenu.gotoPortlet(
-				category = "Security",
-				panel = "Control Panel",
-				portlet = "OAuth 2 Administration");
+		task ("Add CP able.com") {
+			var baseURL = PropsUtil.get("portal.url");
 
-			LexiconEntry.gotoEntryMenuItem(
-				menuItem = "Edit",
-				rowEntry = "Analytics Cloud");
-		}
+			Navigator.openSpecificURL(url = "${baseURL}/group/control_panel/manage?p_p_id=com_liferay_portal_instances_web_portlet_PortalInstancesPortlet");
 
-		task ("Get clientId") {
-			WaitForSPARefresh();
+			PortalInstances.addCP(
+				mailDomain = "www.able.com",
+				virtualHost = "www.able.com",
+				webId = "www.able.com");
 
-			var generatedClientId = selenium.getElementValue("OAuth2#ADMIN_INPUT_FORM_CLIENT_ID");
-			var portalURL = PropsUtil.get("portal.url");
+			PortalInstances.viewCP(virtualHost = "www.able.com");
 		}
 
-		task ("Go to scopes and add com.liferay.oauth2.provider.sample.jar file") {
-			Navigator.gotoBack();
-
-			OAuth2.editScopes(
-				applicationName = "Analytics Cloud",
-				checkGlobalScope = "Everything");
-
-			FileInstall.deployFileOnServer(
-				filePath = "coreinfrastructure/security/oauth2",
-				liferayDependencyJars = "com.liferay.oauth2.provider.sample.jar");
+		task ("Login at able") {
+			User.firstLoginPG(
+				password = "test",
+				userEmailAddress = "test@www.able.com",
+				virtualHostsURL = "http://www.able.com:8080");
 		}
 
-		task ("Assert everything.read is checked in sample-app resource panel") {
-			LexiconEntry.gotoEntryMenuItem(
-				menuItem = "Edit",
-				rowEntry = "Analytics Cloud");
-
-			Navigator.gotoNavItem(navItem = "Scopes");
-
-			OAuth2.expandResourcePanel(resourcePanel = "sample-app");
+		task ("Add Unassign power user role to test") {
```

Comment:
Is it added or removed?
By the macro name, it looks like we actually: "Unassing Power User role from test user"